lib/NMISNG.pm

```diff
@@ -2933,6 +2952,82 @@ sub opstatus_collection
 	return $self->{_db_opstatus};
 }
 
+
+# loads code plugins if necessary, returns the names
+# args: none
+# returns: list of package/class names
+sub validation_plugins
+{
```

Comment:
We just discussed this, fold this into the sub plugin and have everything call that. Apologies for suggesting it needs to be a different function, everything checks "can" before calling so this should be fine